packages/aragon-cli/src/lib/ipfs/config.js

```diff
@@ -43,16 +39,18 @@ const IPFSCORS = [
   },
 ]
 
-export const ensureIPFSInitialized = async () => {
-  if (!getBinary('ipfs')) {
-    throw new Error(
-      'IPFS is not installed. Use `aragon ipfs install` before proceeding.'
-    )
+export const ensureRepoInitialized = async (binPath, repoPath) => {
+  if (!binPath) {
+    throw new Error(NO_INSTALLATION_MSG)
   }
 
-  if (!existsSync(getDefaultRepoPath())) {
+  if (!existsSync(path.resolve(repoPath))) {
     // We could use 'ipfs daemon --init' when https://github.com/ipfs/go-ipfs/issues/3913 is solved
```

Comment:
I think we still need this because we configure the ports before starting the daemon. I don't think that's possible do to using the `ipfs daemon --init` command